contribute-docs/toc.yml

```diff
@@ -0,0 +1,11 @@
+toc:
+  - file: index.md
+  - file: api-docs/index.md
```

Comment:
>  - file: api-docs/index.md

This seems like a duplication of what's in the `api-docs/toc.yml` file. Is the duplication necessary?